git-hub

```diff
@@ -396,8 +417,8 @@ class RequestManager:
 
     # Configure the class to use basic authentication instead of OAuth
     def set_basic_auth(self, username, password):
-        self.basic_auth = "Basic " + base64.urlsafe_b64encode("%s:%s" %
-            (username, password))
+        self.basic_auth = b"Basic " + base64.urlsafe_b64encode(
+                _encode("%s:%s" %(username, password)))
```

Comment:
you might want to fix indentation here and probably keep space after `%`

Reply:
I agree about the `%`, but I believe indentation inside parentheses should be double, as at e.g. [L411](https://github.com/sociomantic-tsunami/git-hub/blob/v1.x.x/git-hub#L411) and [L624](https://github.com/sociomantic-tsunami/git-hub/blob/v1.x.x/git-hub#L624) and various other places. However I also see a bunch of places where it's not double. I'm OK either way :grin: 

> Continuation lines should align wrapped elements either vertically using Python's implicit line joining inside parentheses, brackets and braces, or using a hanging indent [7].
[...]
> `# Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.`

(from [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation).)

Reply:
you're right about the indentation. It might be worth to push a patch to fix the indentation inconsistencies at some point in a separate PR